src/app/grants/page.tsx

```diff
@@ -101,11 +101,9 @@ export default function Grants() {
         </CardGrid>
       </PageSection>
 
-      {grantGraduates.length > 0 && (
-        <PageSection kicker="Past Recipients" title="Grant Graduates">
-          <FeaturedGrantGraduates grantGraduates={grantGraduates} />
-        </PageSection>
-      )}
+      <PageSection kicker="Past Recipients" title="Grant Graduates">
```

Comment:
We can't guarantee these exist - Since filter can return an empty arran. 

```ts
const grantGraduatesSlugs = featuredGrantGraduatePaths.map(
  extractSlugFromFilename,
)

const grantGraduates = ecosystemProjects.filter((item) =>
  grantGraduatesSlugs?.includes(item.slug),
)
```

Try changing the type here to be a none-empty arre and it will be clear what I mean.

```ts
type FeaturedGrantsGraduatesProps = {
  grantGraduates: Array<EcosystemProject>
}
```

Reply:
Ah I see.

Tho Zod will scream at us if the `grantsGraduates` are empty, it could happen somewhere in the process that we pass an empty array.

<img width="1213" alt="Screenshot 2024-12-24 at 09 22 51" src="https://github.com/user-attachments/assets/8c676414-04cb-4ece-8ab9-641c1c1fdae8" />


Thank you, I added the condition for the whole section and non empty type for `FeaturedGrantGraduates` 👍